onnxruntime/test/contrib_ops/attention_op_test_helper.h

```diff
@@ -30,14 +30,16 @@ struct AttentionTestData {
   std::vector<float> bias_data;
   std::vector<float> rel_pos_bias_data;
 
-  std::vector<float> past_key_data;
-  std::vector<float> past_value_data;
-
-  std::vector<float> fp32_output_data;
-  std::vector<float> fp16_output_data;
-
-  std::vector<float> present_key_data;
-  std::vector<float> present_value_data;
+  struct OutputData {
+    std::vector<float> past_key_data;
+    std::vector<float> past_value_data;
+    std::vector<float> fp32_output_data;
+    std::vector<float> fp16_output_data;
+    std::vector<float> present_key_data;
+    std::vector<float> present_value_data;
+  };
+  OutputData biased;      // output by using bias_data
```

Comment:
I like the original design where each data is one test case.

If we want to re-use most inputs, we can do like the following:
```
void GetTestDataOfCase1_no_bias(AttentionTestData& data){
    GetTestDataOfCase1_bias(data);
    data.bias_data.clear(); // remove bias
    data.fp16_output_data = { ...some new data... } ; // update expected output
}
```

Reply:
Adopted.